libs/utils/env.py

```diff
@@ -600,6 +601,14 @@ def _init_target_gem5(self):
             system['platform']['description']))[0]
         self.conf['board'] = self.conf['board'].lower() + suffix
 
+        energy_file = self.conf['board'] + '_energy';
+        try:
+            self.nrg_model = globals()[energy_file]
```

Comment:
Regarding 2:
I didn't look at the core ordering & frequencies - it makes sense in this case to have a different file. As for specifying cores, that could be done in the "real" platform file (it's the case for the HiKey960 https://github.com/ARM-software/lisa/blob/master/libs/utils/platforms/hikey960.json), but if you need a different file to account for different core ordering anyway it's not really relevant.